package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "module",
+  "engines": {
+    "node": ">=18"
+  },
+  "scripts": {
+    "builder": "ts-node builder.ts"
+  },
+  "dependencies": {
+    "@inquirer/prompts": "^1.0.3",
+    "@types/node": "^20.1.1",
+    "dotenv": "^16.0.3",
+    "ethers": "^6.3.0",
+    "ethers-multisend": "^2.4.0",
+    "ts-node": "^10.9.1"
+  },
+  "devDependencies": {
+    "prettier": "^2.8.8",
+    "typescript": "^5.0.4"
+  }
```

Comment:
should we pin dependencies?

Reply:
Versions are actually frozen in `yarn.lock`. If `yarn.lock` is unchanged, then one is guaranteed to install the same versions the project was built for. `package.json` is only here for maintenance, to inform developers about version requirements.